docs/docs/references/users/README.md

```diff
@@ -1,5 +1,95 @@
 # 👨‍👩‍👧‍👦 Users
 
+The _users_ are the main entities of the identity service.
+We will describe the user-related concepts and details in the following.
+
 ## Profile
 
+Each user has a profile containing the user information, more than the name and contact details.
```

Comment:
I would expect `user info` has already contain name and contact details.
`more than the name and contact details` -> something like "including even more details like `social identities`" (haven't checked this rephrase with Grammarly :P)

Reply:
Updated.

```
Each user has a profile containing the user information.
```

Reply:
Updated per offline discussion.

```
Each user has a profile containing the user information.
```